onward/app/controllers/RelatedController.scala

```diff
@@ -24,16 +24,29 @@ object RelatedController extends Controller with Related with Containers with Lo
     val excludeTags = request.queryString.getOrElse("exclude-tag", Nil)
     related(edition, path, excludeTags) map {
       case related if related.items.isEmpty => JsonNotFound()
-      case trails => renderRelated(trails.items.sortBy(-_.content.trail.webPublicationDate.getMillis))
+      case trails => renderRelated(trails.items.sortBy(-_.content.trail.webPublicationDate.getMillis), "related content")
     }
   }
 
-  private def renderRelated(trails: Seq[RelatedContentItem])(implicit request: RequestHeader) = Cached(30.minutes) {
+  /*
+  give the first 2 from each tag excluding the current article
+   */
+  def renderTags(path: String) = MemcachedAction { implicit request =>
+    val edition = Edition(request)
+    val keywordIds = request.queryString.getOrElse("keywordIds", Nil).flatMap(_.split(","))
+
+    getRelatedByTags(edition, path, keywordIds) map {
+      case related if related.items.isEmpty => JsonNotFound()
+      case trails => renderRelated(trails.items.sortBy(-_.content.trail.webPublicationDate.getMillis), "related stories")
```

Comment:
Contradicting my previous suggestion... I changed this to "related stories". Thinking about it again, "related" is appropriate - because surely the point of tags is to gather related things. But "content" is a bland term. And "stories" is consistent with the "more on this story" title we give to story packages. Thus, "related stories".